src/main/java/moomoo/task/Expenditure.java

```diff
@@ -29,8 +32,19 @@ public Expenditure(String name, double cost, LocalDateTime dateTime) {
         this.cost = cost;
         this.dateTime = dateTime;
     }
-    
-    
+
```

Comment:
Is this a test function? Can i remove this code?

Reply:
Yup it's just so I got something to work with. You can remove.